ledger/participant-integration-api/src/main/scala/platform/apiserver/LedgerFeatures.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.apiserver
+
+import com.daml.ledger.api.v1.experimental_features.{
+  CommandDeduplicationFeatures,
+  ExperimentalContractIds,
+}
+
+case class LedgerFeatures(
+    staticTime: Boolean = false,
```

Comment:
What does it mean, in the context of ledger features? Also, do you need it here only to create the `ApiVersionService` or are there other reasons?

Reply:
This class is only used for implementing the `ApiVersionService`.

It is something governed by the ledger, not the participant, so I thought it belongs here.

Reply:
Is it `staticTimeEnabled` or `staticTimeSupported`?

Reply:
It's "enabled", but I am not sure that makes sense. There is nothing stopping a ledger, in principle, from _only_ supporting static time. In that case it would not be "enabled", but simply "time is static".

To that end, I thought `staticTime` was a more useful label.

Reply:
How about `staticTimeMode`? It's less suggestive, compared to `enabled`, that the ledger also supports something else.

Reply:
Ack, GitHub didn't show me your comment until I merged and refreshed the page. That's a good idea and I will update it.

Reply:
Really minor though.